django/db/backends/mysql/features.py

```diff
@@ -67,6 +69,11 @@ def is_sql_auto_is_null_enabled(self):
             return result and result[0] == 1
 
     @cached_property
+    def supports_over_clause(self):
+        """Only MariaDB supports this."""
+        return self.connection.mysql_version >= (10, 2)
```

Comment:
I've got an idea for how we can separate MySQL/MariaDB on the DB backend coming...